src/ElixirLanguageServer.ts

```diff
@@ -28,7 +28,7 @@ export default class ElixirLanguageServer {
       nova.subscriptions.remove(this.languageClient);
     }
 
-    makeServerExecutable();
+    makeServerExecutable(path);
```

Comment:
Since all that path is being used for is an early return I'd prefer to not pass it in to `makeServerExecutable` and instead wrap this in a conditional.

```ts
if (!path || path === "") {
  makeServerExecutable(path);
}
```